Angular2WebpackVisualStudio/src/Angular2WebpackVisualStudio/webpack.prod.js

```diff
@@ -84,18 +85,17 @@ module.exports = {
         }),
         new webpack.optimize.CommonsChunkPlugin(
             {

```

Comment:
You are right but the same applies here :) Lets wait for @robisim74 s opinion about this.